controller-runtime/src/main/kotlin/app/simplecloud/controller/runtime/database/Database.kt

```diff
@@ -9,14 +11,27 @@ class Database(
     fun setup() {
         System.setProperty("org.jooq.no-logo", "true")
         System.setProperty("org.jooq.no-tips", "true")
-        val setupInputStream = Database::class.java.getResourceAsStream("/schema.sql")
-            ?: throw IllegalArgumentException("Database schema not found.")
-        val setupCommands = setupInputStream.bufferedReader().use { it.readText() }.split(";")
-        setupCommands.forEach {
-            val trimmed = it.trim()
-            if (trimmed.isNotEmpty())
-                context.execute(org.jooq.impl.DSL.sql(trimmed))
+        
+        val schemaFile = when (context.dialect().family()) {
+            SQLDialect.MARIADB, SQLDialect.MYSQL -> "/schema-mysql.sql"
```

Comment:
Maybe improve the file structure, add a schema directory and name them like this `schema/mysql.db` or `schema/default.sql`